### PR TITLE
Sonar Action: add quality gate conditions details

### PR DESCRIPTION
**Pull request by sfwn**
result images:
image
image

### CODE REVIEW

File: actions/sonar/1.0/internal/sonar_ce_task.go [hunk 1]
```diff
@@ -91,6 +92,8 @@ func (sonar *Sonar) handleScannerReportTaskFile(filePath string) (*ScannerReport
 			logrus.Infof("invoke sonar to get ce task status failed, continue, err: %v\n", err)
 			continue
 		}
+		fmt.Println("ceTask below:")
+		fmt.Println(jsonparse.JsonOneLine(ceTask))
```
Comment (Effet) [inline actions/sonar/1.0/internal/sonar_ce_task.go:96]: why not using logrus?

File: actions/sonar/1.0/internal/analysis.go [hunk 2]
```diff
@@ -187,15 +180,29 @@ func (sonar *Sonar) Analysis(cfg *Conf) (map[ResultKey]string, error) {
...
+		return results, err
+	}
+	qualityGateStatus := qualityGateResult.Status
+	results.Add(ResultKeyQualityGateStatus, string(qualityGateStatus))
+	// add condition results to meta
+	fmt.Println("quality gate conditions result below:")
+	for _, condition := range qualityGateResult.Conditions {
+		// log
+		fmt.Printf("metric: %s, status: %s\n", condition.MetricKey, condition.Status)
+		fmt.Printf("metric(detail): %s, detail: %s\n", condition.MetricKey, jsonparse.JsonOneLine(condition))
```
Comment (Effet) [inline actions/sonar/1.0/internal/analysis.go:194]: why not using logrus?

**Review by Effet — CHANGES_REQUESTED** [review]

File: actions/sonar/1.0/internal/sonar_ce_task.go (hunk 1 above)
Comment (sfwn) [reply actions/sonar/1.0/internal/sonar_ce_task.go:96]: See the first picture, we already have timestamp provides by spot. No need to use logrus to cause duplicated time info.

File: actions/sonar/1.0/internal/analysis.go (hunk 2 above)
Comment (sfwn) [reply actions/sonar/1.0/internal/analysis.go:194]: same.

**Review by Effet — APPROVED** [review]

**Pull request merged**